File: src/main/java/de/steklopod/sec02/Lec07FluxVsList.java
package de.steklopod.sec02;

import de.steklopod.courseutil.Util;
import de.steklopod.sec02.helper.NameGenerator;

public class Lec07FluxVsList {

    public static void main(String[] args) {

       // List<String> names = NameGenerator.getNames(5);
        //System.out.println(names);

        NameGenerator.getNames(5)
                .subscribe(Util.onNext());



    }

}

File: src/main/kotlin/de/steklopod/sec02/Lec07FluxVsList.kt
package de.steklopod.sec02

import de.steklopod.sec02.helper.NameGenerator
import de.steklopod.utils.Util.onNext

object Lec07FluxVsList {
    @JvmStatic
    fun main(args: Array<String>) {

        // List<String> names = NameGenerator.getNames(5);
        //System.out.println(names);
        NameGenerator.getNames(5)
            .subscribe(onNext())
    }
}
